main.py

```
@@ -152,6 +161,26 @@ async def detect(req: Sentence):
+def get_matches(
+    date: str = Query(..., description="경기 날짜(YYYY-MM-DD)"),
+    user: dict = Depends(get_current_user)
+):
+
+    result = get_match_info_by_date(date)
+    return JSONResponse(content=result)
+
+@app.get("/match/{match-id}")
+def get_match_preview(
+    game_id: str = Query(..., description="네이버 경기 ID (예: 20250601HHNC02025)"),
+    user: dict = Depends(get_current_user)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)